app/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="shortcut icon" href="images/favicon.ico">
+  <meta charset="UTF-8">
+  <title>Salt Machine</title>
+</head>
+<body ng-app='saltMachineApp'>
+  <saltMachine></saltMachine>
+  <console></console>
+  <history></history>
+</body>
```

Comment:
**TODO FOR LOST POINTS**
Take out lines 10-11 and your game should be running fine now.